dotCMS/src/main/java/com/dotcms/timemachine/ajax/TimeMachineAjaxAction.java

```diff
@@ -274,6 +275,7 @@ public void disableJob(HttpServletRequest req, HttpServletResponse resp) throws
 
     }
 
+    @CloseDBIfOpened
```

Comment:
If you are saving, you might want to do WrapInTransaction rather than CloseDb

Reply:
The save is happening in a separate Thread, so let's put the annotation at `Thread.run` method level instead and remove the `finally` block inside the `run` method